projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss

```diff
@@ -42,24 +45,22 @@
       display: flex;
       flex-direction: row;
       width: 100%;
-      border: 1px solid $gray-2;
+      //border: 1px solid $gray-2;
```

Comment:
remove this if not needed?

Reply:
Addressed in the fixing-timeline-card-view PR